corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -404,6 +404,7 @@ <h3 class="caption" data-bind="html: header" tabindex="0"></h3>
 </script>
 
 <script type="text/html" id="address-entry-ko-template">
+  <!-- ARIA live region for highlighted geocoder values, populated via javascript -->
```

Comment:
Thinking about how one would go about finding the corresponding JS code, can we include "role=region" (or however this needs to be written) in the selector so a quick search for that in the code would at least help find it.
or if there is a better way?

Reply:
added `role='region'` in the selector. https://github.com/dimagi/commcare-hq/pull/32637/commits/68b36b8b2c993ccc68f7265900bba3f6712034c7